README.md

```diff
@@ -0,0 +1,22 @@
+# Sylt-lang
+-----------
+Sylt is a statically-checked dynamically typed, reference counted programming
+language made for GameJams.
+
+
+## Why does this exist? Why not use X-lang?
+Pfft! Why not?
+    
+
+## Getting started
+Sylt is written entirely in Rust. Pointing to this Git-repo 
+in your Cargo.toml is the easiet way. Alternatively you can
+build the interpreter and run your own programs from there.
```

Comment:
Maybe some instructions on how to build? Even if it isn't more than "1. git clone 2. cargo build". Also, right now this paragraph contains two ideas.